Hunting Queries/Microsoft 365 Defender/Execution/Bitsadmin Activity.yaml

```diff
@@ -0,0 +1,39 @@
+id: 2458e226-76e6-458c-8bf7-8766cc993b82
+name: Bitsadmin Activity
+description: |
+  Background Intelligent Transfer Service (BITS) is a way to reliably download files from webservers or SMB servers. 
+  This service is commonly used for legitimate purposes, but can also be used as part of a malware downloader. 
+  Additionally, bitsadmin can be used to upload files and therefore can be used for data exfiltration. This
+  query will identify use of bitsadmin.exe for either purpose and will identify directionality file transfer
+  directionality.
+tactics: 
+- Persistence
+- CommandAndControl
+- Exfiltration
+requiredDataConnectors:
+- connectorId: MicrosoftThreatProtection
+  dataTypes:
+  - DeviceProcessEvents
+query: |
+  DeviceProcessEvents
+  | where 
+      (FileName =~ "bitsadmin.exe" or ProcessVersionInfoOriginalFileName =~ 'bitsadmin.exe')
```

Comment:
It appears that ProcessVersionInfoOriginalFileName is not available in the schema that is brought through to Sentinel.  Can you double check your query in Sentinel instance?